uninstall.php

```diff
@@ -5,4 +5,7 @@
 	die;
 }
 
-// silence is golden.
\ No newline at end of file
+global $wpdb;
+
+// delete all options and transients that contain mimi
+$wpdb->query( "DELETE FROM $wpdb->options WHERE option_name LIKE '%mimi-%'" );
```

Comment:
Wondering if this is too general, from a quick scan everything is stored in a single option and otherwise there are transients. My concern is could affect settings for other plugins such as the gravity forms addon and I think there's 1 or 2 others for something called shopp.

Could this be split into a single `delete_option( 'mad-mimi-settings' );` call and then make the db query more specific by changing the `LIKE` to `LIKE '_transient_mimi-%'`?